admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
+
+Migration related to auth_tokens (app passwords)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Upgrading to 10.0.8 includes migrations related to *auth_tokens* (*app passwords*). When users create *app passwords* as separate passwords for their clients the upgrade duration will increase depending on user count. Please consider this when planning the upgrade.
+
+Changed behavior of e-mail autocomplete for public link share dialog
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When the *'Sharing'* settings option ``Allow users to send mail notifications for shared files`` for public links is enabled, users can send public links via mail from within the web interface. The behavior of the autocomplete when entering mail addresses in the public link share dialog has been changed. Previously the autocomplete queried for local users, users from federated address books and contacts from CardDAV/Contacts App. As public links are not intended for sharing between ownCloud users (local/federated), those have been removed. Contacts synchronized via CardDAV or created in the Contacts app will still appear as suggestions.
+
+Global option for CORS domains
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For security reasons ownCloud has a *Same-Origin-Policy* that prevents requests to ownCloud resources from other domains as the domain the backend server is hosted on. If ownCloud resources should be accessible from other domains, e.g. for a separate web frontend operated on a different domain, administrators can now globally specify policy exceptions via *CORS (Cross-Origin Resource Sharing)* using ``'cors.allowed-domains'`` in *config.php*. Please check *config.sample.php* for more information.
+
+Śolved known issues
+~~~~~~~~~~~~~~~~~~~
+
+- Bogus "Login failed" log entries have been removed (see 10.0.7 known issues)
+- The *Provisioning API* can now properly set default or zero quota
+- User quota settings can be queried through *Provisioning API*
+- A regression preventing a user from setting their e-mail address in the settings page has been fixed
+- File deletion as guest user works again (trash bin permissions are checked correctly)
+
+Known issues
+~~~~~~~~~~~~
+
+- Issue with multiple theme apps and Mail Template Editor
+As of ownCloud Server 10.0.5 it is only possible to have one theme app enabled simultaneously. When having a theme app enabled and trying to enable a second one this will result in an error. However, when also having the Mail Template Editor enabled this scenario will lead to the *'General'* settings section for administrators being displayed wrong (https://github.com/owncloud/core/issues/31134). As a remedy administrators can either uninstall the second theme app or disable the Mail Template Editor app.
+
+- ``occ transfer:ownership`` does not transfer public link shares (https://github.com/owncloud/core/issues/31150).
```

Comment:
It does if the share was created by the source user.
It doesn't if the share was created by the target user (reshare)

Reply:
s/When having a theme app enabled and trying to enable a second one this will result in an error./When a theme app is enabled attempts to enable additional ones will result in an error./g

Reply:
thx, corrected

Reply:
@settermjd I had to revert that as with your suggestion the issue is not understandable. It's exactly this scenario that creates the issue.